.github/workflows/tests.yml

```diff
@@ -3,14 +3,20 @@ name: Run tests
 on:
   push:
     branches:
-    - trunk
+      - trunk
+    paths:
+      - "ci/**"
+      - ".github/workflows/tests.yml"
```

Comment:
Test results may change only if we modify a file inside the `ci` directory. Or if we modify this workflow. This is why we can restrict this workflow only to the list of the files above.